sql/slow-query.md

```diff
@@ -8,87 +8,209 @@ category: user guide
 
-```
-2018/08/20 19:52:08.632 adapter.go:363: [warning] [SLOW_QUERY] cost_time:18.647928814s
-process_time:1m6.768s wait_time:12m11.212s backoff_time:600ms request_count:2058
-total_keys:1869712 processed_keys:1869710 succ:true con:3 user:root@127.0.0.1
-txn_start_ts:402329674704224261 database:test table_ids:[31],index_ids:[1],
-sql:select count(c) from sbtest1 use index (k_1)
+```sql
+# Time: 2019-03-18-12:10:19.513961 +0800
+# Txn_start_ts: 407078752230047745
+# User: root@127.0.0.1
+# Conn_ID: 1
+# Query_time: 16.479155653
+# Process_time: 5.634 Wait_time: 0.002 Request_count: 2 Total_keys: 20002 Process_keys: 20000
+# DB: test
+# Index_ids: [1]
+# Is_internal: false
+# Digest: 3635413fe0c8e1aa8307f4f018fe1a9325ea0b97452500106d3f6783fcb65e33
+# Num_cop_tasks: 10
+# Cop_proc_avg: 1 Cop_proc_p90: 2 Cop_proc_max: 3 Cop_proc_addr: 10.6.131.78
+# Cop_wait_avg: 5 Cop_wait_p90: 6 Cop_wait_max: 7 Cop_wait_addr: 10.6.131.79
+# Memory_max: 4096
+select * from t_slim, t_wide where t_slim.c0=t_wide.c0;
 ```
 
 ### Fields description
 
-This section describes fields in the slow query log based on the usage example above.
-
-#### `cost_time`
-
-The execution time of this statement. Only the statements whose execution time exceeds [slow-threshold](../op-guide/tidb-config-file.md#slow-threshold) output this log.
-
-#### `process_time`
-
-The total processing time of this statement in TiKV. Because data is sent to TiKV concurrently for execution, this value might exceed `cost_time`.
-
-#### `wait_time`
-
-The total waiting time of this statement in TiKV. Because the Coprocessor of TiKV runs a limited number of threads, requests might queue up when all threads of Coprocessor are working. When a request in the queue takes a long time to process, the waiting time of the subsequent requests will increase.
-
-#### `backoff_time`
-
-The waiting time before retry when this statement encounters errors that require a retry. The common errors as such include: lock occurs, Region split, the TiKV server is busy.
-
-#### `request_count`
-
-The number of Coprocessor requests that this statement sends.
-
-#### `total_keys`
-
-The number of keys that Coprocessor has scanned.
+* `Time`: Indicates the print time of log.
+* `Txn_start_ts`: Indicates the start timestamp of the transaction, that is, the ID of the transaction. You can use this value to `grep` the transaction-related logs.
+* `User`: Indicates the name of the user who executes this statement.
+* `Conn_ID`: Indicates the Connection ID (session ID). For example, you can use the keyword `con:3` to `grep` the log whose session ID is 3.
+* `Query_time`: Indicates the execution time of this statement. Only the statement whose execution time exceeds slow-threshold output this log (the unit is second). The unit of all the following time fields is second.
+* `Process_time`: The total processing time of this SQL in TiKV. Because the data will be sent to TiKV in parallel, this value may exceed `Query_time`.
```

Comment:
* `Process_time`: The total processing time of this SQL statement in TiKV. Because the cop-task is sent to TiKV execute concurrently. This value may exceed `Query_time`.